docs/docs/api/01-API/table-of-contents.md

```diff
@@ -1,10 +1,11 @@
 ---
 displayed_sidebar: apiSidebar
+custom_edit_url: null
```

Comment:
Please remove this as it breaks the `Edit this page` button at the bottom of the page.